radix-engine/src/system/node.rs

```diff
@@ -30,7 +31,7 @@ pub enum RENodeModuleInit {
     Metadata(MetadataSubstate),
 
     /* Access rules */
-    ObjectAccessRulesChain(MethodAccessRulesChainSubstate),
+    ObjectAccessRulesChain(MethodAccessRulesSubstate),
```

Comment:
Can we rename `ObjectAccessRulesChain` to `ObjectAccessRules`?

Reply:
yup good catch